test/testvector-metafeed-managment.json

```diff
@@ -1,111 +1 @@
adc06c2ac16f76565343a7479706532303a06006d657461666565642f746f6d6273746f6e656536363a0400ae7f51b80958e96f7cb10635c58dd09078451bb3e5098a734bdfc4044a658c45f50013bde38530a7f2dc36977a04cc465ddf73a60d75bc810e647655ded9950e656536363a040041257f5a5fd9bb71b2e8ada464a7b495cc146d5b58439d4685c70a6753664aefd498271ecf189ded3e597727aaf24ccc0532ecd78172121a4efab08134d5ef0265","Key":"%tirbHwMq4LX9MF5nI0bLVcdAK0YzptqOJNOiKGJR3pY=.bbmsg-v1","Author":"@b99R2e7lj8h7NFqGhOu6lCGy8gLxWV+J4ORd1X7rP3c=.bbfeed-v1","Sequence":3,"Previous":"%3n1tMdGwA6acShm4qE5mSKLzlPwn3BKkJ7A6LigIF0A=.bbmsg-v1","Timestamp":0,"HighlevelContent":[{"type":"metafeed/tombstone","subfeed":"@Oo6OYCGsjLP3n+cep4FiHJJZGHyqKWztnhDk7vJhi3A=.ed25519","metafeed":"@b99R2e7lj8h7NFqGhOu6lCGy8gLxWV+J4ORd1X7rP3c=.bbfeed-v1","tangles":{"metafeed":{"root":"%Td7xs4cE/hpabRR+2kWPh65SAjZUf0/DFgrcBsKsFvc=.bbmsg-v1","previous":["%Td7xs4cE/hpabRR+2kWPh65SAjZUf0/DFgrcBsKsFvc=.bbmsg-v1"]}}},{"Name":"subfeed1 signature","HexString":"0400ae7f51b80958e96f7cb10635c58dd09078451bb3e5098a734bdfc4044a658c45f50013bde38530a7f2dc36977a04cc465ddf73a60d75bc810e647655ded9950e"}],"Signature":"040041257f5a5fd9bb71b2e8ada464a7b495cc146d5b58439d4685c70a6753664aefd498271ecf189ded3e597727aaf24ccc0532ecd78172121a4efab08134d5ef02"},{"EncodedData":"6c6c33343a00036fdf51d9eee58fc87b345a8684ebba9421b2f202f1595f89e0e45dd57eeb3f7769346533343a0104b62adb1f032ae0b5fd305e672346cb55c7402b4633a6da8e24d3a2286251de966930656c6431313a66656564707572706f736533303a06006d657461666565642075706772616465206f66206578697374696e67383a6d6574616665656433343a00036fdf51d9eee58fc87b345a8684ebba9421b2f202f1595f89e0e45dd57eeb3f77373a7375626665656433343a0002e31e35f374db8d3038e9139ce548fd166b44f87d9b15555e1a34331b0765642c373a74616e676c657364383a6d6574616665656464383a70726576696f7573323a0602343a726f6f74323a06026565343a7479706532333a06006d657461666565642f6164642f6578697374696e676536363a0400eef616ed1ba7a8a3ad15a9a4e3dcbe7c791346337fc1a7a5c206ad95a0ff504ae3a803390020d3c5f1ba2a50a6d6fcedb0fba99f1324d8a4253613f1a694730e656536363a040009bd0aa9b0618fabda1895f9496dc917b5e2580d37991e578628ee2e6e7a1dc12a5360e4793230fc6fd49a648aa56fd344d25a5238ac368ba9456de75e25760d65","Key":"%xgbmcNOUVRotdVxpqUcCJhC/90uNeusjrw4KANrFvOk=.bbmsg-v1","Author":"@b99R2e7lj8h7NFqGhOu6lCGy8gLxWV+J4ORd1X7rP3c=.bbfeed-v1","Sequence":4,"Previous":"%tirbHwMq4LX9MF5nI0bLVcdAK0YzptqOJNOiKGJR3pY=.bbmsg-v1","Timestamp":0,"HighlevelContent":[{"type":"metafeed/add/existing","feedpurpose":"metafeed upgrade of existing","subfeed":"@4x4183TbjTA46ROc5Uj9FmtE+H2bFVVeGjQzGwdlZCw=.bamboo","metafeed":"@b99R2e7lj8h7NFqGhOu6lCGy8gLxWV+J4ORd1X7rP3c=.bbfeed-v1","tangles":{"metafeed":{"root":null,"previous":null}}},{"Name":"subfeed signature","HexString":"0400eef616ed1ba7a8a3ad15a9a4e3dcbe7c791346337fc1a7a5c206ad95a0ff504ae3a803390020d3c5f1ba2a50a6d6fcedb0fba99f1324d8a4253613f1a694730e"}],"Signature":"040009bd0aa9b0618fabda1895f9496dc917b5e2580d37991e578628ee2e6e7a1dc12a5360e4793230fc6fd49a648aa56fd344d25a5238ac368ba9456de75e25760d"}]}
```

Comment:
Could this JSON be formatted with the same newlines as it had before?

One programmatic way of achieving this is (in Node.js REPL):

```js
JSON.stringify(require('./test/testvector-metafeed-managment.json'), null, 2)
```

then copy-pasting that back into this json file

Reply:
Ah yes, definitely! Thanks for pointing that out. I will need to push the new vector file once it's created and will be sure to pretty-print it beforehand.